src/app/views/realms/view/firesides/RouteRealmViewFiresides.vue

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+import { computed, ref } from 'vue';
+import { RouterLink } from 'vue-router';
+import { Api } from '../../../../../_common/api/api.service';
+import { Fireside } from '../../../../../_common/fireside/fireside.model';
+import AppIllustration from '../../../../../_common/illustration/AppIllustration.vue';
+import {
+	createAppRoute,
+	defineAppRouteOptions,
+} from '../../../../../_common/route/route-component';
+import { Screen } from '../../../../../_common/screen/screen-service';
+import AppFiresideAvatar from '../../../../components/fireside/avatar/AppFiresideAvatar.vue';
+import AppFiresideAvatarBase from '../../../../components/fireside/avatar/AppFiresideAvatarBase.vue';
+import { illNoComments } from '../../../../img/ill/illustrations';
+import { useRealmRouteStore } from '../view.store';
+
+export default {
+	...defineAppRouteOptions({
+		cache: true,
+		lazy: true,
+		deps: {
+			params: ['path'],
+		},
+		resolver: ({ route }) => Api.sendRequest(`/web/realms/firesides/${route.params.path}`),
+	}),
+};
+</script>
+
+<script lang="ts" setup>
+const routeStore = useRealmRouteStore();
+const firesides = ref<Fireside[]>([]);
+
+const gridColumns = computed(() => {
+	if (Screen.isXs) {
+		return 4;
+	} else if (Screen.isSm) {
+		return 5;
+	}
+
+	return 6;
+});
+
+const placeholderCount = computed(() => {
```

Comment:
generally for things that just return I use the shorthand `computed(() => gridColumns.value * 2);` it'll put it in one line then

Reply:
i dont like comments being outside the functions when they refer to a specific line within them. for one liners tho i guess its fine?